src/GoTrueClient.ts

```diff
@@ -105,6 +113,7 @@ export default class GoTrueClient {
     const settings = { ...DEFAULT_OPTIONS, ...options }
     this.inMemorySession = null
     this.storageKey = settings.storageKey
+    this.refreshTokenSyncStorageKey = `${this.storageKey}-rts`
```

Comment:
is it possible to actually storage this information in the same storage used for the session?